src/core/server/metrics/logging/get_ops_metrics_log.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+import numeral from '@elastic/numeral';
+import { LogMeta } from '@kbn/logging';
+import { EcsOpsMetricsEvent } from './ecs';
+import { OpsMetrics } from '..';
+
+/**
+ * Converts ops metrics into ECS-compliant `LogMeta` for logging
+ *
+ * @internal
+ */
+export function getEcsOpsMetricsLog({ process, os }: Partial<OpsMetrics>): LogMeta {
+  const processMemoryUsedInBytes = process?.memory?.heap?.used_in_bytes;
+  const processMemoryUsedInBytesMsg = processMemoryUsedInBytes
+    ? `memory: ${numeral(processMemoryUsedInBytes).format('0.0b')} `
+    : '';
+
+  // ECS process.uptime is in seconds:
+  const uptimeVal = process?.uptime_in_millis
+    ? Math.floor(process.uptime_in_millis / 1000)
+    : undefined;
+
+  // HH:mm:ss message format for backward compatibility
+  const uptimeValMsg = uptimeVal ? `uptime: ${numeral(uptimeVal).format('00:00:00')} ` : '';
+
+  // Event loop delay is in ms
+  const eventLoopDelayVal = process?.event_loop_delay;
+  const eventLoopDelayValMsg = eventLoopDelayVal
+    ? `delay: ${numeral(process?.event_loop_delay).format('0.000')}`
+    : '';
+
+  const loadEntries = {
+    '1m': os?.load ? os?.load['1m'] : undefined,
+    '5m': os?.load ? os?.load['5m'] : undefined,
+    '15m': os?.load ? os?.load['15m'] : undefined,
+  };
+
+  const loadVals = [...Object.values(os?.load ?? [])];
+  const loadValsMsg =
+    loadVals.length > 0
+      ? `load: [${loadVals.map((val: number) => {
```

Comment:
I'm not sure formatting belongs here. The metrics plugin might define formatting `pattern` for its logging context. The main benefits: 
- this logic is lazily called only when logging for the context is enabled
- end-users might customize `pattern` for the `metics` context

Reply:
I added the formatting here to have the message mimic the message from the legacy logger. @restrry are you suggesting we leave all the formatting out for the whole log message and replace it with a stringified version of the meta or just for the load values?

Reply:
We could replace the array formatting here with one that looks like:
`memory: 235.1MB uptime: 0:00:32 load: {'1m': 29.83, '5m': 29.83, '15m': 29.83} delay: 0.171`

The message formatting is a little more work:
```
const loadEntriesMsg = `load: {'1m': ${
    os?.load ? numeral(os?.load['1m']).format('0.00') : undefined
  }, '5m': ${os?.load ? numeral(os?.load['1m']).format('0.00') : undefined}, '15m': ${
    os?.load ? numeral(os?.load['1m']).format('0.00') : undefined
  }} `;
```

Reply:
I actually suggested define a custom pattern for `metrics.ops` context as done here
https://github.com/elastic/kibana/blob/2b5071ff36da10af6b9bd40864700d36d30e51ec/x-pack/plugins/security/server/audit/audit_service.ts#L222-L240
for this we need to specify a custom pattern: `[%date][%level][%logger] %{meta.process.memory.heap.used_in_bytes}....` I forgot that we don't support key [lookup](https://logging.apache.org/log4j/2.x/manual/lookups.html) for `meta` map yet. ignore my comment.